stock_helper_delivery/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Stock Helpers - Delivery",
+    "summary": "Add methods shared between various stock+delivery modules",
+    "version": "13.0.1.0.0",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Hidden",
+    "depends": ["stock", "delivery", "stock_helper"],
```

Comment:
should it really depend on stock_helper?

Reply:
Not really, I was re-using the tests class from `stock_helper` but we could remove this dependency if it's really needed

Reply:
Updated to get rid of this dependency + increase test coverage